source/_docs/drupal-cache.md

```diff
@@ -6,8 +6,25 @@ categories: [drupal]
 ---
 To maximize your site's performance on Pantheon and to take advantage of our Varnish caching, you'll need to configure your site's performance settings.
 
-## Drupal 8
+## Drupal 8 Performance Configuration
 
+Visit `/admin/config/development/performance` for Drupal's performance settings.
+
+### Caching
+![Drupal 8 Caching options](/source/docs/assets/images/d8-cache-config.png)  
+**This is a key setting**. It determines what value Drupal delivers in its `max-age` header, which is how long the reverse-proxy layer will retain a cache.
+
+Performance is often a trade-off between how fresh your content is, and how fast you want to deliver it to the internet. A good value to start with is 15 minutes, but this is something to consider. If you can set it to an hour, that's great for performance. More than a day is usually excessive, since the edge cache will decay over that amount of time in most cases.
+
+On Pantheon, this value defaults to 15 minutes. This is done on the first cache-clear operation on the site; immediately after installing the site, you may see this set to `&lt;no caching&gt;`. In this case, press the "Clear all caches" button, or select the page cache maximum age from the available selections.
```

Comment:
Character entities wrapped in code styling are not converted to characters, suggest `<no caching>` instead.